interface.go

```diff
@@ -88,6 +88,8 @@ type Config struct {
 	// MaxReceiveConnectionFlowControlWindow is the connection-level flow control window for receiving data.
 	// If this value is zero, it will default to 1.5 MB for the server and 15 MB for the client.
 	MaxReceiveConnectionFlowControlWindow protocol.ByteCount
+	// Keepalive defines whether keep-alive is enabled.
+	Keepalive bool
```

Comment:
This should be camel-case: `KeepAlive`. Sorry for being pedantic here, but since this is part of the public API, changing this later will be hard.